workflow/snakemake_rules/select_strains.smk

```diff
@@ -70,6 +70,27 @@ rule join_metadata:
             --output {output.metadata:q} 2>&1 | tee {log}
         """
 
+# Annotate strains in the metadata that have "GIHSN" in the strain name to
+# indicate whether it was collected as part of the Global Influenza Hospital
+# Surveillance Network (GIHSN)
+rule annotate_metadata_with_gihsn:
+    input:
+        metadata="data/{lineage}/metadata_joined.tsv",
+    output:
+        metadata="data/{lineage}/metadata_with_gihsn.tsv",
```

Comment:
do you want to `temp()` this?

Reply:
(hah, you're comment popped up right when I clicked merge)

Yeah, adding `temp()` is probably best here.

Reply:
> (hah, you're comment popped up right when I clicked merge)

Achievement unlocked: approve PR after it is already merged 🙌

Reply:
Updated in https://github.com/nextstrain/seasonal-flu/pull/200